src/pages/Projects/ProjectDetail/ProjectDetail.tsx

```diff
@@ -96,8 +96,9 @@ export function ProjectDetail({
         />
       </Flex>
       <RoleList
-        createRole={createRole}
         project={project}
+        createRole={createRole}
+        updateRole={updateRole}
```

Comment:
If you're just passing all 3 of these in as props and not using them elsewhere, why not move the hook call into the component itself?

Reply:
yep makes sense